Content/0.3.2 - Account Settings/accountSettings.json

```diff
@@ -53,4 +55,19 @@
 	"accountSettings_behaviors_removableByAnyone": "Anyone can remove this asset from accounts and dApps.",
 	"accountSettings_behaviors_nftDataChangeable": "Data that is set on these NFTs can be changed.",
 	"accountSettings_behaviors_nftDataChangeableByAnyone": "Anyone can change data that is set on these NFTs."
+
+	"accountSettings_debug_turnIntoDappDefinition": "Turn into dApp Definition account type",
```

Comment:
three instances of this..?

Reply:
it's a very important key, better safe than sorry

Reply:
I also need to go through the files and see if there are any json errors, missing commas etc. Will do that now.